3_javascript/javascript1/week2/javascript.js

```diff
@@ -0,0 +1,99 @@
+//Flight booking fullname function
+
+function getFullname(firstname, surname, gender, useFormalName = false) {
+    if (!firstname.trim() && !surname.trim()) {
+        return "Invalid name";
+    }
+    if (gender === "male") {
```

Comment:
I think, to be more precise here, you would need to wrap your if's into one more if, where you will check if `useFormalName` is true or false :
`
  if (useFormalName) {
    if (gender === "male") {
      return "Lord " + firstname + " " + surname;
    } else if (gender === "female") {
      return "Lady " + firstname + " " + surname;
    } else {
      return firstname + surname;
    }
  }`